.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: Build Book
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - run: bundle install
+      - run: rake build:CI
+      - uses: actions/github-script@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const fs = require('fs').promises;
+
+            const { repo: { owner, repo }, sha } = context;
+
+            const release = await github.repos.createRelease({
+              owner, repo,
+              tag_name: process.env.GITHUB_REF,
```

Comment:
I put version in metadata of book (on first lines in [rails6/fr/api_on_rails.adoc](https://raw.githubusercontent.com/madeindjs/api_on_rails/3ddfd34ecd1940c5bb1f0cd94b7693496dbe11c1/rails6/fr/api_on_rails.adoc) per exampe). Do you think it's possible to get tag name from a version file in repository ?

Reply:
Yeah, that's easy. 
I believe that tag_name must be unique. So how do you plan on making it unique on every push to master? 
Do you think it is a good idea to append a unique number (e.g. [GITHUB_RUN_ID](https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables)) to the tag name?

Reply:
Alternatively, we can consider using GITHUB_RUN_NUMBER. 
 
![image](https://user-images.githubusercontent.com/36763164/91636851-233a1800-ea04-11ea-85e7-a35cdfca3fc2.png)

Reply:
So maybe we can do this ?

```suggestion
              tag_name: `v6.${process.env.GITHUB_RUN_NUMBER}`,
```

What do you think about ?

Reply:
I like it. I will commit your suggestion.